Fabric.Authorization.AccessControl/src/app/modules/access-control/access-control-routing.module.ts

```diff
@@ -2,17 +2,17 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
 
-import { UserlistComponent } from './userlist/userlist.component';
-import { UseraddComponent } from './useradd/useradd.component';
+import { MemberListComponent } from './member-list/member-list.component';
+import { MemberAddComponent } from './member-add/member-add.component';
 
 const routes: Routes = [
   {
```

Comment:
Does path need to be set here as well?

Reply:
Do you mean for the member list route? No it does not need to be able to because it is the default route for this router. The main router will route to this module and this router will load the member list component.